xml_parser.py

```diff
@@ -0,0 +1,22 @@
+#This script is a basic xml parser with simple xinclude support
```

Comment:
This isn't the functionality I was imagining. Cyclus' native xml parser should provide the only xml parsing necessary.